backend/plugins/frontline_api/src/modules/integrations/facebook/meta/automation/utils/messageUtils.ts

```
@@ -67,13 +71,38 @@ export const generatePayloadString = (
+      buttons: TBotConfigMessageButton[];
+    }[];
+    quickReplies: {
+      _id: string;
+      text: string;
+      image_url?: string;
+    }[];
+    image: string;
+  },
 ) => {
-  let botData: any[] = [];
+  let botData: TBotData[] = [];
```

ESLint

[error] 'botData' is never reassigned. Use 'const' instead.

(prefer-const)